index.ts

```diff
@@ -382,6 +406,15 @@ export class OAuth2AuthCodePKCE {
         this.state.refreshToken = refreshToken;
       }
 
+      if (explicitlyExposedTokens) {
+        tokensToExpose = explicitlyExposedTokens.reduce(
+          (a: ObjStringDict, token: string) =>
+            json[token] ? { ...a, [token]: json[token] } : a,
+          {}
+        );
```

Comment:
I think this code could more easily be understood using .map().filter(). Could you please change it to use those? While *I* understand it, I can easily see someone wanting to glance over this .reduce().

Like so:

```
tokensToExpose = Object.fromEntries(explicitlyExposedTokens
  .map((token: string) => [token, json[token]])
  .filter((property: object | undefined) => property !== undefined));
```

I think it's clear to see this is much easier to understand. :slightly_smiling_face: 

(Also I realize you used .reduce() to transform the array into a dictionary/object, but there's https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/fromEntries which can easily handle this.)